### PR TITLE
Spacecat 561 jumphost operations

### PR DESCRIPTION
**Pull request by piobig2871**

### CODE REVIEW

File: steps/steps.py [hunk 1]
```diff
@@ -72,9 +72,10 @@ def then_i_should_be_able_to_list_networks(context):
     def then_i_should_be_able_to_create_a_network(context, network_name: str):
         network = context.client.network.find_network(name_or_id=network_name)
         assert network is None, f"Network with {network_name} already exists"
-        context.client.network.create_network(name=network_name)
-        assert context.client.network.find_network(
-            name_or_id=network), f"Network called {network_name} created"
+        network = context.client.network.create_network(name=network_name)
+        # time.sleep(5)
```
Comment (piobig2871) [inline steps/steps.py:76]: remove sleep

File: steps/steps.py [hunk 2]
```diff
@@ -94,3 +95,26 @@ def then_i_should_be_able_to_create_a_network(context, jumphost_name: str):
         # context.client.network.delete_network(server)
         # assert context.client.network.find_network(
```
Comment (piobig2871) [inline steps/steps.py:96]: uncomment and fix functionality

File: steps/steps.py [hunk 3]
```diff
@@ -94,3 +95,26 @@ def then_i_should_be_able_to_create_a_network(context, jumphost_name: str):
...
+        subnets = context.client.network.subnets()
+        assert subnets, "Failed to list subnets. No subnets found."
+
+    @then('I should be able to create a subnet with name {subnet_name} in network {network_name} with {cidr}')
+    def then_i_should_be_able_to_create_a_subnet(context, subnet_name: str, network_name: str, cidr):
+        network = context.client.network.find_network(name_or_id=network_name)
+        assert network is not None, f"Network with name {network_name} does not exist"
+        subnet = context.client.network.create_subnet(name=subnet_name, network_id=network.id, ip_version=4,
+                                                      cidr=cidr)
+        # time.sleep(5)
```
Comment (piobig2871) [inline steps/steps.py:111]: remove sleep

File: steps/steps.py [hunk 4]
```diff
@@ -2,7 +2,7 @@
 import yaml
 from behave import given, when, then
 import openstack
-import libs.loggerClass as log
+import time
```
Comment (piobig2871) [inline steps/steps.py:5]: not needed

**Review by Erik-Kostelansky-dNation — APPROVED** [review]

**Pull request merged**